src/SocialAuthUserManager.php

```diff
@@ -490,7 +507,7 @@ public function loginUser(User $drupal_user) {
   protected function nullifySessionKeys() {
     if (!empty($this->sessionKeys)) {
       array_walk($this->sessionKeys, function ($session_key) {
-        $this->session->set($session_key, NULL);
+        $this->session->set($this->dataHandler->getSessionPrefix() . $session_key, NULL);
```

Comment:
I'm still confused with this. I still think I might be making a mistake. Please, can you check if this part is correct?